modules/acs-architecture-connections.adoc

```diff
@@ -0,0 +1,58 @@
+// Module included in the following assemblies:
+//
+// * architecture/acs-architecture.adoc
+:_mod-docs-content-type: CONCEPT
+[id="acs-architecture-connections_{context}"]
+= {product-title-short} connection protocols and default ports
+
+The protocols and default ports for connections within {product-title-short} and between {product-title-short} and external components are shown in the following table.
+
+.{product-title-short} connections between components
+
+[cols="3",options="header"]
+|===
+| Component or external entity | Connection type | Port
+
+| Central and external services, for example, Jira and PagerDuty
+| HTTPS
+| ?
+
+| Central and secured cluster
+| HTTPS
+| 443
+
+| Central and sensor
+| TCP
+| 443
+
+| Central and user ({product-title-short} web portal or REST API)
+| HTTPS
+| 443
+
+| Central and user (CLI)
+| gRPC
```

Comment:
this is mostly true, though I believe there is a setting which forces HTTPS traffic (probably uncommonly used, so may not be worth mentioning)

Reply:
I still think it's worth checking with the Merlin team(?) to see if we should mention HTTPS here, too

Reply:
@kcarmichael08 As I can see from `roxctl --help` - we have:
```
     --force-http1                Always use HTTP/1 for all connections (advanced; only use if you encounter connection issues). Alternatively, enable by setting the ROX_CLIENT_FORCE_HTTP1 environment variable to true (default false)
```
This ticket: https://issues.redhat.com/browse/ROX-5787
And changelog for v51: https://github.com/stackrox/stackrox/blob/master/CHANGELOG.md#510

Maybe you can change it in doc, similar to `Central and Sensor on secured cluster` section:
- gRPC
- HTTPS (with `--force-http1` option)